ExpenseMaster.Tests/Services/IncomeServiceTests.cs

```diff
@@ -0,0 +1,167 @@
+﻿using AutoMapper;
+using ExpenseMaster.BusinessLogic.Dto;
+using ExpenseMaster.BusinessLogic.Implementations;
+using ExpenseMaster.BusinessLogic.Interfaces;
+using ExpenseMaster.DAL.Models;
+using Moq;
+using System.Linq.Expressions;
+using Xunit;
+
+namespace ExpenseMaster.Tests.Services
+{
+    public class IncomeServiceTests
+    {
+        private readonly Mock<IRepositoryWrapper> _mockRepositoryWrapper;
```

Comment:
Why do you write like here only in this file? Why others like this
![image](https://github.com/Ilya57061/ExpenseMaster/assets/118177215/4c852308-6daa-4adf-ae4e-ad8ff9e78558)